doc/bitwise_shift.rst

```diff
@@ -0,0 +1,30 @@
+The BitwiseShift Extension
+===================
+
+The BitwiseShift extensions provides the following operators:
+
+* ``shl``
+* ``shr``
```

Comment:
Why textual and not `>>` or `<<`. IMHO `and`, `or` and co is intuitive, but `shl` not.